src/Sylius/Bundle/PromotionBundle/Form/Type/ActionType.php

```diff
@@ -29,7 +27,7 @@ class ActionType extends AbstractConfigurationType
     public function buildForm(FormBuilderInterface $builder, array $options = [])
     {
         $builder
-            ->add('type', 'sylius_promotion_action_choice', [
+            ->add('type', ActionChoiceType::class, [
```

Comment:
I don't think we want to couple to this specific class. Removing type names in Symfony 3.0 is a blocker for us because we allow for easy substitution of the form class by overriding the name and also use one class for multiple entities [see](https://twitter.com/pjedrzejewski/status/673838287312654336). This change should not be introduced without prior discussion.